test/assets/frames/one-frame-url-fragment.html

```diff
@@ -0,0 +1 @@
+<iframe src='./frame.html&test=fragment'></iframe>
```

Comment:
Shouldn't this be `./frame.html?test=fragment` or `./frame.html#fragment` instead?
According to https://github.com/puppeteer/puppeteer/blob/main/new-docs/puppeteer.protocol.page.frame.urlfragment.md the fragment is `#fragment` and `?param=value` is often referred to as query.

Reply:
@campersau That's a good point. Happy to accept a PR that changes these names!